scripts/votemanager/VoteManagerControls.tsx

```diff
@@ -79,7 +84,8 @@ export default class VoteManagerControls extends Component<any, any> {
         <div className="btn-group btn-group-block">
           <button className="btn btn-secondary" id="intro-restore-btn" onClick={() => this.resetSelectedDelegates()}>Restore</button>
           <button className="btn btn-secondary" id="intro-unvote-btn" onClick={() => this.wipeSelectedDelegates()}>Unvote All</button>
-          <button className="btn btn-secondary tooltip" id="intro-optimize-btn" data-tooltip="Vote for the highest paying delegates" onClick={() => this.setSelectedToOptimized()}>Vote For Maximum Payouts</button>
+          <button className="btn btn-secondary tooltip" id="intro-optimize-btn" data-tooltip="Vote for the delegates contributing to Lisk" onClick={() => this.setSelectedToContrib()}>Vote For Contributors</button>
+          <button className="btn btn-secondary tooltip" id="intro-optimize-btn" data-tooltip="Vote for the delegates currently forging" onClick={() => this.setSelectedToMaximum()}>Vote For Forgers</button>
```

Comment:
Id should be unique:

```
id="intro-optimize-btn"
```

this is also used in VoteManagerIntro.tsx, please add a record for it so that during the introduction of the vote manager this button is also explained.

Reply:
Oh yeah, I keep skipping the intro, heh.